<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>411</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Open Sans</family>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Aplikasi Pendeteksi Masker - Register User</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	background-color: rgb(0, 23, 37);
	color: rgb(255, 255, 255);
}

QPushButton{
	background-color: rgb(0, 0, 255);
}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Open Sans SemiBold</family>
     <pointsize>18</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(73, 131, 241);</string>
   </property>
   <property name="text">
    <string>Tambah User Baru</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="usernameText">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>90</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Open Sans</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
  border-radius: 10px;
  color: rgb(255, 255, 255);
  border: 2px solid;
  background-color:  rgb(64, 67, 66);
  padding: 0px 0px 0px 15px;
}

QLineEdit:hover{
  border: 2px solid rgb(255, 255, 255)
}

QLineEdit:focus{
  border: 2px solid rgb(73, 131, 241)
}</string>
   </property>
   <property name="placeholderText">
    <string>username</string>
   </property>
  </widget>
  <widget class="QPushButton" name="signUpButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>320</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Open Sans SemiBold</family>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
  border-radius: 7px;
  background-color: rgb(0, 0, 255);
  color: rgb(255, 255, 255)
}

QPushButton:hover{
   border: 2px solid rgb(0, 0, 255);
   color: rgb(0, 0, 255);
   background-color: rgb(0, 23, 37);
}</string>
   </property>
   <property name="text">
    <string>Buat Akun Baru</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordText">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>160</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Open Sans</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
  padding-left: 5px;
  border-radius: 10px;
  color: rgb(255, 255, 255);
  border: 2px solid;
  background-color:  rgb(64, 67, 66);
  padding: 0px 0px 0px 15px;
}

QLineEdit:hover{
  border: 2px solid rgb(255, 255, 255)
}

QLineEdit:placeholdertext{
  color: rgb(255, 0, 0);
  padding: 5px, 0x, 0px, 0px;
}

QLineEdit:focus{
  border: 2px solid rgb(73, 131, 241)
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::EchoMode::Password</enum>
   </property>
   <property name="cursorPosition">
    <number>0</number>
   </property>
   <property name="placeholderText">
    <string>password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordConfirmText">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>230</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Open Sans</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
  padding-left: 5px;
  border-radius: 10px;
  color: rgb(255, 255, 255);
  border: 2px solid;
  background-color:  rgb(64, 67, 66);
  padding: 0px 0px 0px 15px;
}

QLineEdit:hover{
  border: 2px solid rgb(255, 255, 255)
}

QLineEdit:placeholdertext{
  color: rgb(255, 0, 0);
  padding: 5px, 0x, 0px, 0px;
}

QLineEdit:focus{
  border: 2px solid rgb(73, 131, 241)
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::EchoMode::Password</enum>
   </property>
   <property name="cursorPosition">
    <number>0</number>
   </property>
   <property name="placeholderText">
    <string>konfirmasi password</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="image/image_resource.qrc"/>
 </resources>
 <connections/>
</ui>
